lib/src/checks.dart

```diff
@@ -1,4 +1,51 @@
+import 'package:nyxx/nyxx.dart';
 import 'package:nyxx_commands/nyxx_commands.dart';
 import 'package:running_on_dart/src/settings.dart';
 
 final administratorCheck = UserCheck.anyId(adminIds, 'Administrator check');
+
+final connectedToAVoiceChannelCheck = Check((IContext context) async {
+  final selfMember = await context.guild!.selfMember.getOrDownload();
+
+  if (selfMember.voiceState == null || selfMember.voiceState!.channel == null) {
+    await context.respond(MessageBuilder.content('I have to be in a voice channel to use this command'));
```

Comment:
Responses should be handled in the error handler - the methods for interacting with a context from within checks will be removed in a future version of nyxx_commands.